documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -0,0 +1,654 @@
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    char1 >= char2              |    char1 >= char2                             |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    ord(char)                   |    as(<integer>, char)                        |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    chr(n)                      |    as(<character>, n)                         |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    char.upper()                |    as-uppercase(char)                         |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    char.lower()                |    as-lowercase(char)                         |
++--------------------------------+-----------------------------------------------+
+
+Strings
+^^^^^^^
++--------------------------------+-----------------------------------------------+
+| Python                         | Dylan                                         |
++================================+===============================================+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    isinstance(obj, str)        |    instance?(obj, <string>)                   |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    char * k                    |    make(<string>, size: k, fill: char)        |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | N/A                                           |
+|                                |                                               |
+|    "".join([char for char in   |                                               |
+|         [char, ...]])          |                                               |
```

Comment:
What's the purpose of this example?  To create a string from a sequence of characters in Dylan is just `as(<string>, #['a', 'b', 'c'])`  Maybe the Python side should just be `"".join(['a', 'b', 'c'])`